loculus_values/values.yaml

```
@@ -193,19 +197,22 @@ defaultOrganismConfig: &defaultOrganismConfig
         noInput: true
         columnWidth: 80
         order: 90
+        orderOnDetailsPage: 1000
       - name: earliestReleaseDate
         displayName: Earliest release date
-        header: Sample details
+        header: Submission details
         type: date
         rangeSearch: true
         noInput: true
         includeInDownloadsByDefault: true
         columnWidth: 95
         order: 90
+        orderOnDetailsPage: 2950
```

```suggestion
        orderOnDetailsPage: 5400
```

(I won't push this immediately to avoid possibly resetting the preview, but please apply before merging - it doesn't need repreviewing after - it will move the earliest release date to bottom of submission details.